CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ add_subdirectory(${MBED_ROOT})
 
 add_executable(${APP_TARGET})
 
+mbed_os_configure_app_target(${APP_TARGET})
```

Comment:
we use `mbed_` prefix, not `mbed_os`, this should be fixed on CMake branch, not part of this PR - consistency

Reply:
Ah good point.